db/models.py

```diff
@@ -46,3 +51,68 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name="orders",
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        creation_time = self.created_at.strftime("%Y-%m-%d %H:%M:%S")
+        return creation_time
```

Comment:
There is no need to allocate a variable to store the output of the `strftime` here, return it directly.